backend/apps/api/src/routes/image.py

```
@@ -1,13 +1,125 @@
+from src.models.user import User
 
 router = APIRouter()
 
+def get_user_by_id(user_id: int, db: Session) -> User | None:
+    """사용자 ID로 조회 (내부용)"""
+    return db.query(User).filter(User.id == user_id).first()
+
+
+@router.get("/images/public")
+async def list_all_images(
+    db: Session = Depends(get_db),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)